Kernel/API/POSIX/unistd.h

```diff
@@ -38,6 +38,7 @@ enum {
     _SC_PAGESIZE,
     _SC_GETPW_R_SIZE_MAX,
     _SC_CLK_TCK,
+    _SC_SYMLOOP_MAX /* The number of symbolic links that can be traversed in the resolution of a pathname in the absence of a loop. */
```

Comment:
We don't have any comments for the other entries. Again, maybe a link to the relevant specification would be more suitable.

Reply:
Removed this comment altogether